src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -120,6 +131,8 @@ def generate_weighted_strategy_pool():
   # We consider mutator plugin separately as it is only supported by a small
   # number of fuzz targets and should be used heavily when available.
   if do_strategy(strategy.MUTATOR_PLUGIN_STRATEGY):
+    logs.log()
     pool.add_strategy(strategy.MUTATOR_PLUGIN_STRATEGY)
-
+  
+  logs.log('Weighted strategy pool was generated.')
```

Comment:
nit: log with more details?

Reply:
just wanted to make sure it was running down that code path. im deleting all the log statements that i added previously